addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -588,11 +594,14 @@
 
         <t t-foreach="category_data" t-as="category">
             <div class="mb-2" t-if="(category['slides'] or channel.can_publish) and (search_category and search_category.id == category['id'] or not search_category)">
-                <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['name'] != 'Uncategorized'">
-                    <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
+                <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
```

Comment:
Why move this up ?

Reply:
The first time I've moved this, it was for test something but I reverted the change and forgot to replace it. But finally, I had to move it there to apply the good style to the h5 in all cases.